docs/01-engage/02-accounts/03-contact-details/01-enable-contact-preferences/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Enable Contact Preferences
+sidebar_position: 1
+---
+
+Contact is simply a way that involves meeting or communicating with anyone. So, when a contact wants to receive communication by any means, whether it is by telephone or email we call it a **contact preference**. This is important in terms of knowing and maintaining a valuable relationship with the donor or sponsor (the one you are contacting). 
```

Comment:
This intro needs re-writing, doesn't work well. It needs to be something like:

Contact preferences track the different ways of communicating with someone (telephone, post, email, SMS) called channels, and the different purposes (fundraising, marketing, administration etc.) called categories. Both can be configured by your system administrator to meet your organization's specific needs.

Maintaining accurate contact preferences is important not only to ensure a good relationship with supporters (such as by opting them out of postal mailers if they have requested) but is also important in many countries for complying with data protection laws.